Packs/Base/ReleaseNotes/1_15_1.md

```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
 ##### CommonServerPython
-- Maintenance and stability enhancements.
\ No newline at end of file
+- Maintenance and stability enhancements.
```

Comment:
Why this change?

Reply:
It's an inner change of the way the `IndicatorsSearcher` class works, but I'll change it to be something more infomercial.

Reply:
probably got picked up when merging from master. Reverting